projects/dendrites/permutedMNIST/experiments/__init__.py

```diff
@@ -36,10 +35,10 @@
 # Collect all configurations
 CONFIGS = dict()
 CONFIGS.update(BASE)
+CONFIGS.update(CENTROID)
+CONFIGS.update(HYPERPARAMETERSEARCH)
 CONFIGS.update(BATCH)
```

Comment:
1. note alphabetical order,
2. the line `CONFIGS.update(CENTROID)` is now repeating (a few lines later), so I don't think you need to add it here.